index.html

```diff
@@ -9,42 +9,57 @@
 </head>
 
 <body onload="loadGame()">
-    <section id="output">
-        <ul class="bananas">
-            <li class="letter hidden" id="letter-0">_</li>
-            <li class="letter hidden" id="letter-1">_</li>
-            <li class="letter hidden" id="letter-2">_</li>
-            <li class="letter hidden" id="letter-3">_</li>
-            <li class="letter hidden" id="letter-4">_</li>
-            <li class="letter hidden" id="letter-5">_</li>
-            <li class="letter hidden" id="letter-6">_</li>
-            <li class="letter hidden" id="letter-7">_</li>
-            <li class="letter hidden" id="letter-8">_</li>
-            <li class="letter hidden" id="letter-9">_</li>
-        </ul>
-    </section>
-
-    <section id="stage">
-        This is where the stage will go to show player progress
-    </section>
+    <header>
+        <h1>Guess or Die Game!</h1>
+    </header>
+    <main>
+        
+        <section id="output">
+            <ul class="bananas">
+                <li class="letter hidden" id="letter-0">_</li>
+                <li class="letter hidden" id="letter-1">_</li>
+                <li class="letter hidden" id="letter-2">_</li>
+                <li class="letter hidden" id="letter-3">_</li>
+                <li class="letter hidden" id="letter-4">_</li>
+                <li class="letter hidden" id="letter-5">_</li>
+                <li class="letter hidden" id="letter-6">_</li>
+                <li class="letter hidden" id="letter-7">_</li>
+                <li class="letter hidden" id="letter-8">_</li>
+                <li class="letter hidden" id="letter-9">_</li>
+            </ul>
+        </section>
+        
+        <section>
+                <h3>Guessed Letters</h3>
```

Comment:
Make sure to check that child elements are properly nested in their respective parent elements, and only indented by one. Lines 38-42 are also a bit off.